content/https/https-5/contents.lr

```diff
@@ -0,0 +1,12 @@
+_model: question
+---
+title: Why should you use Tor instead of your regular browser with DoH?
+---
+seo_slug: browser-with-doh-or-tor
+---
+description:
+DoH doesn't guarantee actual privacy. 
+
+Centralised DoH is currently a privacy net negative since anyone that could see your metadata can still see your metadata when DNS is moved to a third party. Additionally, that third party then gets a complete log per device of all DNS queries, in a way that can even be tracked across IP addresses.

```

Comment:
This also seems like an overstatement, at least the first part.  If your network provider is your adversary (not an uncommon model for people interested in Tor), the network provider does not necessarily learn the metadata held by the third party.

The points about per-device metadata detection and tracking are potentially correct, and also potentially mitigatable with the right implementation (something Tor could actually lead on, in terms of avoiding session ticket reuse, cookies, etags, etc), so it's a shame to have them mixed up with this spurious "net negative" allegation.